docs/build-docs.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+rm -rf build/
+
+# Move theming elements into the docs folder
+cp -R pytket-docs-theming/_static .
```

Comment:
For the other extensions I've added a `docs/README.md` file linking to the pytket-docs-theming repo instructions

https://github.com/CQCL/pytket-qir/blob/main/docs/README.md

Totally optional